_slides/13-shansen3030.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Scott is starting to "git" this!
```

Comment:
Awesome!  I would like $1,000 dollars haha

Reply:
Thanks for the Approval.   $1000.00 check is in the mail.  👍